### DISCUSSION TITLE
CI setup

### DISCUSSION THREAD

**Discussion by lispyclouds** [qvacua/vimr#854] [Ideas]
Hey @qvacua 👋🏽 

Thanks a lot for all your hard work on VimR and I'm one of the very very happy users of it and using it as my daily driver for more than 2 years now!

With the 0.5 release of neovim, I wanted to know if there is a setup where we can get nightly builds for VimR too, tracking the upstream maybe. I looked around the source and the only thing i could see is this [file](https://github.com/qvacua/vimr/blob/develop/ci/create_build_job.groovy).

I wanted to know if there is something like Github Actions or CircleCI setup in place or plans for it? If not I am happy to help setup something like that automatically tagging, releasing changes using maybe Github Actions. At least I would be really happy if we have a CI making builds and I think it would help out someone else too! 😄

**Comment by qvacua** [comment]
I'm sorry for the extreme late reply; somehow I missed this entry. One could build a GitHub action which merges Neovim's master. Occasionally, the job would fail since there could be merge conflicts, but that would be OK and can be fixed manually. There are two issues:
1. GitHub actions do not yet support Apple Silicon builds, i.e., no universal binary is possible.
2. Signing and notarizing the resulting App is not possible. I know there are ways to do this, but I haven't yet looked into it and I'm not sure whether I want to...
3. I have to find time to implement it 😅: Getting a GitHub action to work is time-consuming, esp. when some Git actions are involved...
